plop-templates/component/package.json.hbs

```diff
@@ -14,11 +14,11 @@
     "build:cjs": "babel --root-mode upward src --out-dir dist/cjs",
     "build:css": "build-css --useGlamor -i dist/cjs/css/index.js",
     "build:js": "cross-env ESM=true babel --root-mode upward src --out-dir dist/esm",
-    "build:watch": "npm run build:js -- --watch",
-    "prepublish": "npm run build",
+    "build:watch": "yarn build:js -- --watch",
```

Comment:
no ` run` required here, right?

Reply:
right. yarn gets rid of all that run silliness